lib/web_ui/flutter_js/src/canvaskit_loader.js

```diff
@@ -5,12 +5,12 @@
 import { createWasmInstantiator } from "./instantiate_wasm.js";
 import { joinPathSegments } from "./utils.js";
 
-export const loadCanvasKit = (deps, config, browserEnvironment, canvasKitBaseUrl) => {
-  if (window.flutterCanvasKit) {
-    // The user has set this global variable ahead of time, so we just return that.
-    return Promise.resolve(window.flutterCanvasKit);
-  }
-  window.flutterCanvasKitLoaded = new Promise((resolve, reject) => {
+export const loadCanvasKit = async (deps, config, browserEnvironment, canvasKitBaseUrl) => {
```

Comment:
`loadCanvasKit` doesn't need to be `async`, does it?

Reply:
Ah, you're right.